### DISCUSSION TITLE
Headline Hyperlinks

### DISCUSSION THREAD

**Discussion by jgollenz** [nvim-orgmode/orgmode#360] [Q&A] [answered]
Do [these lines](https://github.com/nvim-orgmode/orgmode/blob/72a7a62530e110bd999e9b55d23308d7dbeb39a4/DOCS.md?plain=1#L746-L747) refer to headlines within the same file or can they be in a different file? If the latter is the case, it doesn't seem to work for me.

**Comment by kristijanhusak** [comment, marked as answer]
Within the same file. Initially it was across multiple files, but emacs export complained so we removed it. You can achieve something similar with file path (example: `file:/path/to/myfile.org::*My headline`)

**Reply by jgollenz** [reply]
Thanks, I'll make that more explicit in the docs then :+1:
